index.js

```diff
@@ -1,76 +1,106 @@
-/* jshint node: true */
 'use strict';
 
 var path = require('path');
 var Funnel = require('broccoli-funnel');
 var mergeTrees = require('broccoli-merge-trees');
+var coverageMiddleware = require('./lib/coverage-middleware');
+var bodyParser = require('body-parser');
+
+function logErrors(err, req, res, next) {
+  console.error(err.stack);
+  next(err);
+}
+
 module.exports = {
-    name: 'Ember CLI Blanket',
```

Comment:
reformatted the whole file - we've got 4 spaces and 2 spaces etc, etc - needs a cleanup overall